src/style/rStats.css

```diff
@@ -1,80 +1,72 @@
 .rs-base {
-  position: absolute;
-  z-index: 10000;
-  padding: 10px;
-  background-color: #ed134a;
-  width: 300px;
-  font: 300 10px monospace;
+  background-color: #EF2D5E;
+  border-radius: 0;
+  font: 10px monospace;
   left: 5px;
-  top: 5px;
+  line-height: 1.2em;
+  opacity: 0.75;
   overflow: hidden;
-  opacity: 0.85;
-}
-
-.rs-base h1 {
-  margin: 0;
-  padding: 0;
-  font-size: 1.4em;
-  font-weight: 300;
-  margin-bottom: 5px;
-  color: #111;
-  cursor: pointer;
-}
-
-.rs-base div.rs-group {
-  margin-bottom: 10px;
-}
-
-.rs-base div.rs-group:last-child {
-  margin-bottom: 0;
+  padding: 10px;
+  position: fixed;
+  top: 5px;
+  width: 300px;
+  z-index: 10000;
 }
 
-.rs-base div.rs-group.hidden {
+.rs-base.hidden {
   display: none;
 }
 
-.rs-base div.rs-fraction {
-  position: relative;
-  margin-bottom: 5px;
-}
-
-.rs-base div.rs-fraction p {
-  width: 70px;
-  text-align: right;
-  margin: 0;
+.rs-base h1 {
+  color: #fff;
```

Comment:
it's bigger, has margin, and is not constrained to the columns. the stats shouldn't draw too much attention from the scene